charts/enterprise-logs/small.yaml

```diff
@@ -0,0 +1,115 @@
+# Requirements to ingest ~11MB/s with 30 days retention
+
+admin_api:
+  replicas: 3
+  resources:
+    limits:
+      cpu: 200m
+      memory: 256Mi
+    requests:
+      cpu: 10m
+      memory: 64Mi
+
+compactor:
+  replicas: 1
+  persistentVolume:
+    size: 100Gi
+  resources:
+    limits:
+      cpu: 4
+      memory: 2Gi
+    requests:
+      cpu: 2
+      memory: 1Gi
+
+consul:
```

Comment:
Can ditch this key, we should remove all references to Consul and just use memberlist from the off.